frontend/packages/shared/src/styles/abstracts/_variables.scss

```diff
@@ -1,20 +1,20 @@
-$color-primary: #fff;
+$color-primary: #ffffff;
 $color-primary-light: #f5f5f5;
-$color-primary-dark: #ccc;
+$color-primary-dark: #cccccc;
```

Comment:
Missä tämä oli käytössä? En löytänyt tätä mistään, ja koodissakin tämä taitaa olla käytössä vain CircularStepperin tyyleissä kun taas Figmassa taitaa olla siinä käytössä `$color-secondary` (jos oikein onnistuin paikallistamaan mistä väristä on kyse).

Reply:
Taisin tehdä tämän osalta vain oletuksen, että sen pitäisi vastata material UI konffien väriä `const primaryDarkColor = '#CCCCCC';`. Mutta jos tätä käytetään paraikaa CircularStepperissä väärin, niin voinen vaihtaa sinne oikean värin $color-secondary käyttöön ja poistaa tämän muuttujan $color-primary-dark.

Reply:
CircularStepperissä tämä $color-primary-dark itse asiassa on kyllä Figmassa speksattu olemaan taustavärillä aktiivista steppiä tulevissa stepeissä. Näiltä osin pitäisi olla ihan oikein näin.